cmd/axelard/cmd/vald/evm/evm.go

```diff
@@ -133,7 +133,7 @@ func (mgr Mgr) ProcessDepositConfirmation(e tmEvents.Event) (err error) {
 		return true
 	})
 
-	v, err := packEvents(events)
+	v, err := packEvents(chain, events)
```

Comment:
`vald` currently votes on an empty event even if it failed to retrieve the tx receipt from the RPC node. Now that a chain maintainer can be deregistered with a tighter threshold for voting incorrectly, I think vald shouldn't submit a vote in this case so as to fall in the unresponsive category.

Comments about this approach:
- Before, core would clear rewards for such cases, but now it won't occur until we implement the change of processing all votes at the timeout.

But I still think it's worth switching due to the harsher penalties otherwise.

Reply:
I suggest doing that later in a separate PR.

Reply:
#1458